.rubocop.yml

```diff
@@ -30,7 +30,12 @@ Sorbet/FalseSigil:
 Sorbet/TrueSigil:
   Enabled: true
   Include:
-  - "**/*.rb"
+    - "**/*.rb"
+  Exclude:
+    - "**/*.rake"
```

Comment:
What made us need to exclude rake files?

Reply:
None of the `.rake` files were typed, and in this bump they started getting flagged for missing `# typed: true`.